redisinsight/ui/src/constants/storage.ts

```diff
@@ -35,7 +35,8 @@ enum BrowserStorageItem {
   capability = 'capability',
   aiChatSession = 'aiChatSession',
   selectedAiChat = 'selectedAiChat',
-  generalChatAgreements = 'generalChatAgreements'
+  generalChatAgreements = 'generalChatAgreements',
+  hideCopilotSplashScreen = 'hideCopilotSplashScreen',
```

Comment:
I would suggest to combine these storage items to one (copilot) with object value